src/storage/resourceLoader.ts

```diff
@@ -319,6 +334,68 @@ export class CCloudResourceLoader extends ResourceLoader {
     return Array.from(registryByEnvId.values());
   }
 
+  /**
+   * Return the topics present in the cluster. Will also correlate with schemas
```

Comment:
Not necessary, but this could be helpful for extra clarity:
```suggestion
   * Return the topics present in the {@link CCloudKafkaCluster}. Will also correlate with schemas
```